addons/mail/static/src/js/chatter.js

```diff
@@ -180,6 +180,7 @@ var Chatter = Widget.extend({
      */
     _closeAttachments: function () {
         if (this.fields.attachments) {
+            this.$el.removeClass('o_chatter_composer_active');
```

Comment:
This introduces UI glitches when clicking toggling off attachment box while composer is open, then toggling composer mode.

I would remove this line from the diff, in addition to the corresponding `addClass('o_chatter_composer_active')` when opening attachment box.